nebu/models/binder.py

```diff
@@ -0,0 +1,68 @@
+from lxml import etree
+from cnxepub.models import (
+    Binder as BaseBinder,
+    DocumentPointer,
+    TranslucentBinder,
+)
+
+
+COLLECTION_ID_TAG = '{http://cnx.rice.edu/mdml}content-id'
+COLLECTION_TAG = '{http://cnx.rice.edu/collxml}collection'
+SUBCOLLECTION_TAG = '{http://cnx.rice.edu/collxml}subcollection'
+MODULE_TAG = '{http://cnx.rice.edu/collxml}module'
+TITLE_TAG = '{http://cnx.rice.edu/mdml}title'
+COLLXML_TOPIC_TAGS = (
+    COLLECTION_TAG,
+    SUBCOLLECTION_TAG,
+    MODULE_TAG,
+    TITLE_TAG,
+)
+
+
+class Binder(BaseBinder):
+    # def __init__(self, *args, **kwargs):
+    #     super(Binder, self).__init__(*args, **kwargs)
+
+    @classmethod
+    def from_collection_xml(cls, filepath):
+        """\
+        Given a ``collection.xml`` as ``filepath``. This provides the same
+        interface as :class:`cnxepub.models.Binder`, but can be created from a
+        ``collection.xml`` file.
+
+        :param filepath: location of the ``collection.xml`` file
+        :type filepath: :class:`pathlib.Path`
+        :return: Binder object
+        :rtype: :class:`Binder`
+
+        """
+        with filepath.open('rb') as fb:
+            elm_iterparser = etree.iterparse(
+                fb,
+                events=('start', 'end'),
+                tag=COLLXML_TOPIC_TAGS,
+                remove_blank_text=True,
+            )
+
+            binder = parent_node = current_node = cls(None)
+            chain = [binder]
+            for event, elm in elm_iterparser:
+                if elm.tag == TITLE_TAG and event == 'start':
+                    if isinstance(current_node, DocumentPointer):
+                        continue
```

Comment:
What do we if the document title is different from the document title in the binder? There is such a thing right...? :thinking:

Reply:
That's a good point! And cnx-epub does take care of this for us by allow us to set "title overrides". Though, I'm not doing that here, so that's a flaw.

Reply:
The question is... is it going to be fixed somewhere else or does it not matter so we can just ignore it? :D ...

Reply:
fixed... And this was because I was working too late at night. Because you can see in the test file I put a placeholder for testing title overrides. 😛